### PR TITLE
extract media form for content type to media admin bundle

### PR DESCRIPTION
**Pull request by nicolasThal**

### CODE REVIEW

File: MediaAdminBundle/DependencyInjection/OpenOrchestraMediaAdminExtension.php [hunk 1]
```diff
@@ -30,5 +30,32 @@ public function load(array $configs, ContainerBuilder $container)
...
+    {
+        $fieldTypes = array_merge(
+            $container->getParameter('open_orchestra_backoffice.field_types'),
+            array(
+                'orchestra_media' => array(
+                    'label' => 'open_orchestra_media_admin.form.field_type.custom_type.media',
+                    'type' => 'orchestra_media',
+                    'options' => array(
+                        'required' => array(
+                            'default_value' => false,
```
Comment (itkg-ngilain) [inline MediaAdminBundle/DependencyInjection/OpenOrchestraMediaAdminExtension.php]: @itkg-alavieille : this is related to your current work, isn't it ?

Comment (itkg-alavieille) [inline MediaAdminBundle/DependencyInjection/OpenOrchestraMediaAdminExtension.php]: yes, in my PR (), i changed the type orchestra_media  by
```
        orchestra_media:
            label: open_orchestra_backoffice.form.field_type.custom_type.media
            type: orchestra_media
            default_value:
                type: orchestra_media
                options:
                    label: open_orchestra_backoffice.form.field_type.default_value
                    required: false
            options:
                required:
                    default_value: false
```

Comment (nicolasThal) [inline MediaAdminBundle/DependencyInjection/OpenOrchestraMediaAdminExtension.php]: allright, i will do the modification before closing this pr :)

File: MediaAdminBundle/DependencyInjection/OpenOrchestraMediaAdminExtension.php [hunk 2]
```diff
@@ -30,5 +30,39 @@ public function load(array $configs, ContainerBuilder $container)
...
+     *
+     * @param ContainerBuilder $container
+     */
+    protected function addMediaFieldType(ContainerBuilder $container)
+    {
+        $fieldTypes = array_merge(
+            $container->getParameter('open_orchestra_backoffice.field_types'),
+            array(
+                'orchestra_media' => array(
+                    'label' => 'open_orchestra_backoffice.form.field_type.custom_type.media',
```
Comment (itkg-alavieille) [inline MediaAdminBundle/DependencyInjection/OpenOrchestraMediaAdminExtension.php]: it's open_orchestra_media_admin.form.field_type.custom_type.media

**Comment by itkg-nanne** [comment]
:+1:

**Comment by nicolasThal** [comment]
@itkg-alavieille update done

**Comment by itkg-alavieille** [comment]
:+1:

**Pull request merged**